ui/raidboss/data/06-ew/trial/zodiark-ex.ts

```diff
@@ -19,6 +124,307 @@ const triggerSet: TriggerSet<Data> = {
+          if (sigil.x > 100)
+            dir1 = 'east';
+
+          if (sigil.y < 100)
+            dir2 = data.sigilMap[sigil.id] === 'Blue' ? 'north' : 'south';
+
+          if (sigil.y > 100)
+            dir2 = data.sigilMap[sigil.id] === 'Red' ? 'north' : 'south';
+        }
+
+        if (!dir1 || !dir2) {
+          console.error(`Exoterikos 1: Unknown layout, ${JSON.stringify(data.storedSigils)}`);
+          data.storedSigils = [];
+          return;
+        }
+
+        data.storedSigils = [];
+        return output[dir2 + dir1]!();
+      },
+      outputStrings: directionOutputStrings,
+    },
+    {
+      id: 'ZodiarkEx Algedon NE/SW',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '67EC', source: 'Zodiark' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '67EC', source: 'Zodiark' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '67EC', source: 'Zordiarche' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '67EC', source: 'ゾディアーク' }),
+      alertText: (_data, _matches, output) =>
+        output.combo!({
+          dir1: output.northeast!(),
+          dir2: output.southwest!(),
+        }),
+      outputStrings: directionOutputStrings,
+    },
+    {
+      id: 'ZodiarkEx Algedon NW/SE',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '67ED', source: 'Zodiark' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '67ED', source: 'Zodiark' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '67ED', source: 'Zordiarche' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '67ED', source: 'ゾディアーク' }),
+      alertText: (_data, _matches, output) =>
+        output.combo!({
+          dir1: output.northwest!(),
+          dir2: output.southeast!(),
+        }),
+      outputStrings: directionOutputStrings,
+    },
+    {
+      id: 'ZodiarkEx Paradeigma 1',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '67BF', source: 'Zodiark' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '67BF', source: 'Zodiark' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '67BF', source: 'Zordiarche' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '67BF', source: 'ゾディアーク' }),
+      condition: (data) => data.paradeigmaCounter === 0,
+      delaySeconds: 0.1,
+      alertText: (data, _matches, output) => {
+        ++data.paradeigmaCounter;
+        return output.underQuetz!();
+      },
+      outputStrings: {
+        underQuetz: {
+          en: 'Under NW Quetzalcoatl',
+        },
+      },
+    },
+    {
+      id: 'ZodiarkEx Paradeigma 2',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '67BF', source: 'Zodiark' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '67BF', source: 'Zodiark' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '67BF', source: 'Zordiarche' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '67BF', source: 'ゾディアーク' }),
+      condition: (data) => data.paradeigmaCounter === 1,
+      delaySeconds: 0.1,
```

Comment:
Oh wow, 0.1?

Reply:
There's some wonky timing issues here because the `=== 0` trigger will fire and then resolve before this one fires, so then this one fires for the same mechanic.